alert/search.go

```diff
@@ -60,6 +60,7 @@ var searchTemplate = template.Must(template.New("search").Parse(`
 	{{end}}
 	{{ if .Search }}
 		AND (
+			CAST(a.id AS TEXT) = substring(:search FROM '\%(.*?)\%') OR
```

Comment:
Maybe we can pass this as a `NamedArg` as a `NullInt64` rather than doing regex at query-time -- that way the DB can skip the check altogether if it's not valid and not need to cast either.

Query would become:
```sql
a.id = :searchID
```

And in `QueryArgs` we could do something like:
```go
var searchID sql.NullInt64
if i, err := strconv.ParseInt(opts.Search, 10, 64); err == nil {
  searchID.Valid = true
  searchID.Int64 = i
}
```